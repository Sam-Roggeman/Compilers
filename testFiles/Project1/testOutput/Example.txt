#PreOrder before optimize
Program,int a0,1,int b0,2,int c0,+,int a0,1,int b0,2
#Preorder after optimalizations
Program,int a0,1,int b0,2,int c0,+,int a0,1,int b0,2
