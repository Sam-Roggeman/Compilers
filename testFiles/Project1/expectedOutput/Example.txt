#PreOrder before optimize
Program,int a0,1,int b0,2,int c0,+,int a0,1,int b0,2
#Preorder after optimalizations
Program,const int a0,1,const int b0,2,const int c0,3
